api/ocm/extensions/download/config/type.go

```diff
@@ -82,8 +84,8 @@ of preconfigured download handler registrations (see <CMD>ocm ocm-downloadhandle
     type: ` + ConfigType + `
     description: "my standard download handler configuration"
     handlers:
-      - name: oci/artifact
-        artifactType: ociImage
+      - name: ` + ocirepo.PATH + `
+        artifactType: ` + artifacttypes.OCI_IMAGE + `
```

Comment:
@open-component-model/maintainers - Is it a bug or done by intention, that downloaders and uploaders use different namings?

Reply:
What do you mean?

Reply:
the uploader/blobhandler is:

```yaml
- name: ocm/ociArtifacts
  artifactType: ociArtifact
```

vs. download

```yaml
- name: oci/artifact
  artifactType: ociImage
```

spot the difference

Reply:
the second also refers to the legacy artifactType "ociImage"

Reply:
but it's the ONLY option for **download** and `oci/artifact` **!=** `ocm/ociArtifacts` (uploader/blobhandler)

maybe there is a reason why it is different for up-/downloaders?